docs/web/1.http/http-status.md

```diff
@@ -1,32 +1,52 @@
 ---
 id: http-status
-title: HTTP 状态码
-sidebar_label: HTTP 状态码
+title: HTTP 响应状态码
+sidebar_label: HTTP 响应状态码
 ---
 
-## 常见状态码
+目前HTTP在Web开发中被广泛使用，在Web开发中不同的HTTP响应状态码往往有不同的含义。在普通网站开发、分布式集群开发、团队协作方面，如果采用了HTTP作为组件之间交互的协议，遵守通用的响应状态码是很有必要的（一是有充分的信息量、二是避免歧义）。
+
+## 响应状态码分类
+HTTP响应状态共有5大类，分别是数字1、2、3、4、5开头。
+
+* 1XX：信息性状态码，表示接收的请求正在处理。
+* 2XX：成功状态码，表示请求正常处理完毕。
+* 3XX：重定向状态码，表示需要进行附加操作以完成请求。
+* 4XX：客户端错误状态码，表示服务器无法处理请求。
+* 5XX：服务器错误状态码，表示服务器处理请求出错。
+
+## 常见的具体状态码
 
 | 大分类 | 状态码 | 含义 |
 | --- | --- | --- |
+| 1XX | 100 Continue | 说明收到了请求的初始部分，请客户端继续。发送了这个状态码之后，服务器必须在请求完成后发送一个最终的响应 |
+|  | 101 Switching Protocols | 说明服务器正在根据客户端的指定，将协议切换成Update 首部所列的协议 |
 | 2XX | 200 OK | 分三种，有两种是强缓存 |
-|  | 201 Created | 请求成功，并创建了一个新的资源，常用在 put 和 post 请求 |
-|  | 202 Accepted | 请求已经接收到，但还未响应，没有结果 |
-|  | 204 No Content | 请求成功，没有内容返回 |
-|  | 206 Partial Content | 请求成功，返回部分内容 |
+|  | 201 Created | 请求成功，并创建了一个新的资源，响应的实体主体部分中应该包含各种引用了已创建的资源的URL，Location 首部包含的则是最具体的引用。（REST里面，POST或者PUT常常返回这样的状态） |
+|  | 202 Accepted | 请求已经接收到，但还未响应，没有结果。这只是意味着接受请求时，它看起来是有效的 |
+|  | 204 No Content | 请求成功，没有内容返回。通常在更新服务器上资源的时候成功的情况下不要求返回实体内容就会产生这个状态码 |
+|  | 206 Partial Content | 请求成功，返回部分内容。往往表示成功执行了一个部分或Range（范围）请求 |
 |  |  |  |
-| 3XX | 301 Moved Permanently | 永久重定向 |
-|  | 302 Found | 临时重定向 |
-|  | 304 Not Modified | 请求的资源并没有被修改，可以直接使用浏览器中缓存 |
+| 3XX | 301 Moved Permanently | 永久重定向，表示请求的资源被分配了新的URI，以后应使用资源现在所指的URI |
+|  | 302 Found | 临时重定向，表示请求的资源被分配了新的URI，希望用户本次使用新的URI访问 |
+|  | 303 See Other | 表示请求对应的资源存在这另一个URI，应使用GET方法定向获取请求的资源 |
+|  | 304 Not Modified | 表示客户端发送附带条件的请求时，服务器端允许请求访问资源，但未满足条件（附带条件指：If-Match、If-Modified-Since、If-None-Match、If-Range、If-Unmodified-Since中任一首部）。304状态码返回时不包含响应的主体部分，304表明了客户端中所请求资源的缓存仍然是有效的，也就是说该资源从上次缓存到现在并没有被修改过。条件请求可以在确保客户端的资源是最新的同时避免因每次都请求完整资源给服务器带来的性能问题。另一种情况是，如果服务器认为客户端缓存的资源已经过期了，那么服务器就会返回HTTP/200 OK响应，响应体就是该资源当前最新的内容。客户端收到200响应后，就会用新的响应体覆盖掉旧的缓存资源。 |
```

Comment:
`304 Not Modified` is explained in detail in this section `/docs/web/1.http/http-cache`, you can refer to this section with some links

Reply:
> `304 Not Modified` is explained in detail in this section `/docs/web/1.http/http-cache`, you can refer to this section with some links

OK.